VERSION

```diff
@@ -1 +1 @@
-1.0.7
+1.0.8
```

Comment:
Please do not update the cookbook's release version as there might be more changes coming up to bundle into a release version bump